src/index.ts

```diff
@@ -83,6 +83,7 @@ export { default as Map, MapProps } from './components/Map';
 export { default as Navbar, NavbarProps } from './components/Navbar';
 export { default as Link, LinkProps } from './components/Link';
 export { default as List, ListProps } from './components/List';
+export * from './components/Popover';
```

Comment:
nit-pick: can we keep the exports explicit?
```suggestion
export { default as Popover, PopoverProps, PopoverOptions } from './components/Popover';
```